sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/enums.py

```diff
@@ -8,6 +8,7 @@
 class KeyWrapAlgorithm(str, Enum):
     """Key wrapping algorithms"""
 
+    aes_256 = "A256KW"
```

Comment:
I'm not 100% sure that we don't need A128KW or A192KW, you should double check what storage will be using, and if this is subject to change.

Reply:
I'm told we need only A256KW. Subject to change is a good question, any insight @rakshith91 or @tg-msft?

Reply:
It's documented as only AES 256 at https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption and hasn't change since it was introduced.  I do not expect this to change any time soon.